src/autocomplete/EmojiProvider.tsx

```diff
@@ -130,6 +132,15 @@ export default class EmojiProvider extends AutocompleteProvider {
             sorters.push(c => c._orderBy);
             completions = sortBy(uniq(completions), sorters);
 
+            completions = completions.slice(0, LIMIT);
+
+            //do a second sort to place emoji matching with frequently used one on top
+            sorters.length = 0;
```

Comment:
```suggestion
            sorters = [];
```

This is more readable, IMO

Reply:
I agree with you, but sorters is a constant

Reply:
Then I would suggest making it into a `let` (or create a new variable)